c_common/front_end_common_lib/local.mk

```diff
@@ -101,14 +101,15 @@ CFLAGS += -DAPPLICATION_NAME_HASH=0x$(APPLICATION_NAME_HASH)
 # Add the modified directories to the include path
 CFLAGS += $(foreach d, $(sort $(SOURCE_DIRS)), -I $(call modified_dir,$(d)))
 
-# Sort out the set of dictionary files needed
-MODIFIED_DICT_FILES := $(foreach d, $(sort $(SOURCE_DIRS)), $(call modified_dir, $(d))log_dict.dict)
-LOG_DICT_FILES += $(wildcard $(SPINN_DIRS)/lib/*.dict)
-LOG_DICT_FILES += $(MODIFIED_DICT_FILES)
-APP_DICT_FILE = $(APP_OUTPUT_DIR)$(APP).dict
-
 # default rule based on list ALL_TARGETS so more main targets can be added later
-ALL_TARGETS += $(APP_DICT_FILE) $(APP_OUTPUT_DIR)$(APP).aplx
+ALL_TARGETS += $(APP_OUTPUT_DIR)$(APP).aplx
+
+$(foreach d, $(sort $(SOURCE_DIRS)), \
+    $(eval ALL_MODIFIES_DIRS += $(call modified_dir, $(d))))
+
+ALL_TARGETS += $(APP_OUTPUT_DIR)$(APP).aplx
+all: $(ALL_MODIFIES_DIRS) $(ALL_TARGETS)
```

Comment:
Is this meant to be here or is this left over from something previous?

Reply:
Without the all: $(ALL_MODIFIES_DIRS) it was not always running the python to build the modified_src.

In master that was triggered by the need for the log dicts but those are now gone.

Reply:
In that case, should the all below be removed?

Reply:
second "all" removed